FileSystem/FileIO.js

```diff
@@ -254,4 +258,122 @@ function printToFile(fileNumber, outputList) {
 
   // Print new line
   printNewline(file);
-}
\ No newline at end of file
+}
+
+/**
+ * Emulates VBA Line Input statement API
+ * @param {number} fileNumber File number
+ * @param {VbaBox} variable reference variable to store line
+ */
+function lineInputFile(fileNumber, variable) {
+  if (!(fileNumber in this.openFiles)) {
+    throw new Error('File Number: ' + fileNumber + ' is not open');
```

Comment:
Are we going to use custom exceptions? we could use the common FileIOException class  that @muskangarg98 is creating

Reply:
Yes we will use Custom Exceptions. @muskangarg98 and I will discuss this. Will send a separate PR for all errors module wise.